data/pidgin/frontpage/index.json

```diff
@@ -1560,6 +1560,24 @@
             "timestamp": 1528978068000,
             "type": "link"
           },
+          {
+            "name": "Select your best ever African 11 players for World Cup",
```

Comment:
Currently we have a story with the same title. Could you add something like `- Video` to make it different and easy to spot while viewing this on localhost.